src/tests/Login/actions/login.test.js

```diff
@@ -68,8 +68,11 @@ describe('Test suite for login action', () => {
     });
 
     const expectedActions = [
-      { type: AUTHENTICATION_FAILED, payload: { errorMessage: 'Passwords did not match.' }, loading: false },
-    ];
+      {
```

Comment:
Why was this change made? Was it part of the issue scope?

Reply:
@kmunene Thanks for this, the line was not really removed, It was changed instead. This was due to linting(The line was too long[103]). I was fixing the tests which were not passing due login tests.